roles/prometheus/templates/targets/zookeeper_exporter_target.json.j2

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "targets": [
+      {%- for item in groups.zookeeper -%}
+      {% if not loop.last %}
+      "{{ hostvars[item].alt_hostname | default(hostvars[item].hostname) }}:9381",
```

Comment:
```suggestion
      "{{ hostvars[item].alt_hostname | default(hostvars[item].hostname) }}:9381"{% if not loop.last %},{% endif %}
```

How about this? Make it clear what the difference is. Also if you don't go this route you need to change the 'else' below:

```diff
-  "{{ hostvars[item].alt_hostname }}:9381"
+ "{{ hostvars[item].alt_hostname | default(hostvars[item].hostname) }}:9381"
```

Reply:
This one seems much cleaner. thanks